uisdk/src/main/java/tech/dojo/pay/uisdk/presentation/ui/result/PaymentResultScreen.kt

```diff
@@ -69,11 +72,21 @@ internal fun ShowResultSheetScreen(
         )
     val coroutineScope = rememberCoroutineScope()
     val state = viewModel.state.observeAsState().value ?: return
+    val view = LocalView.current
```

Comment:
Could we have a shared function to reuse for both this and uisdk/src/main/java/tech/dojo/pay/uisdk/presentation/ui/virtualterminalcheckout/VirtualTerminalCheckOutScreen.kt?

Reply:
done